packages/textlint/package.json

```diff
@@ -59,6 +59,7 @@
     "textlint-rule-no-todo": "^2.0.0",
     "textlint-rule-preset-ja-spacing": "^2.0.0",
     "textlint-rule-preset-jtf-style": "^2.3.0",
+    "textlint-fixer-formatter": "^1.0.0",
```

Comment:
Yay.

We can remove textlint-fixer-formatter's dependencies form `textlint`'s `dependencies` field`.
For example, we can remove chalk from textlint-self

Reply:
I removed `chalk`, `diff`, `string-width` and `text-table` from textlint !